src/ui/pages/Security/AutoLockTimer.tsx

```diff
@@ -39,51 +50,84 @@ export const AUTO_LOCK_TIMER_OPTIONS_TITLES = Object.fromEntries(
 );
 
 export function AutoLockTimer() {
-  const { globalPreferences, setGlobalPreferences } = useGlobalPreferences();
-  const value = globalPreferences?.autoLockTimeout;
+  const { globalPreferences, setGlobalPreferences, mutation } =
+    useGlobalPreferences();
+  useBackgroundKind({ kind: 'white' });
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    if (mutation.isSuccess) {
+      navigate(-1);
+    }
+  }, [mutation.isSuccess, navigate]);
```

Comment:
why not use `onSuccess` callback?

Reply:
Do we have onSuccess callback in useGlobalPreferences? Or can I add a param to the hook?